src/contracts/mainnet/AaveV2wETHIRPayload.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title wETH Interest Rate Curve - Ethereum v2
+ * @author Llama
+ * @notice Upgrade wETH interest rate on Ethereum v2 to match Ethereum v3 Liquidity Pool.
+ * Governance Forum Post: https://governance.aave.com/t/arfc-weth-wsteth-interest-rate-curve-ethereum-network/11372
```

Comment:
The Governance Forum post : https://governance.aave.com/t/arfc-weth-wsteth-interest-rate-curve-ethereum-network/11372/6 also talks about wETH and wstETH changes on Ethereum V3. Are those not meant to be a part of this proposal payload?

Reply:
Those were done as listings when Ethereum v3 launched.
Here's the wstETH v3 interest rate for example: https://etherscan.io/address/0x7b8Fa4540246554e77FCFf140f9114de00F8bB8D#readContract

and ETH: https://etherscan.io/address/0x53F57eAAD604307889D87b747Fc67ea9DE430B01#readContract